types/ws/index.d.ts

```diff
@@ -180,6 +174,11 @@ declare class WebSocket extends EventEmitter {
 }
 
 declare namespace WebSocket {
```

Comment:
Is it possible to add `WebSocketServer` as alias for `Server` inside this namespace? I think something like this

```js
class WebSocketServer extends Server {}
```

should work. I'm not sure if there are better options, and ofc it will only work with `ws@>=8.2.0`.

Reply:
I will add the line after `ws@8.2.0` published.

Reply:
`ws@8.2.0` is out. `WebSocket.WebSocket` should also be added.

Reply:
Thanks.